docs/libraries/libraries-with-cli.md

```diff
@@ -0,0 +1,591 @@
+
+Say you wish to support older frameworks, such as .NET Framework 4.5.2, so you can reach a maximum number of developers and projects.  From a project targeting perspective, this is quite simple!  You will need to specifically target the version of .NET Framework with its relevant Target Framework Monikers.  These are mapped out here:
+
+```
+.NET Framework 2.0   --> net20
+.NET Framework 3.0   --> net30
+.NET Framework 3.5   --> net35
+.NET Framework 4.0   --> net40
+.NET Framework 4.5   --> net45
+.NET Framework 4.5.1 --> net451
+.NET Framework 4.5.2 --> net452
+```
+
+For example, you may need to support .NET Framework 4.0 and .NET Framework 4.5.2, but wish to use some of the newer APIs available in .NET Core and .NET Framework 4.6.  Setting up your `project.json` file like this will do the trick:
+
+```javascript
+{
+    "frameworks":{
+        "net40":{},
+        "dotnet55":{
+            "dependencies":{
+                "System.Runtime":"4.0.0-rc1-*"
+            }
+        }
+    }
+}
+```
+
+And that's it!  Note that you only need to target the lowest version of .NET Framework here.  This is assuming you don't have the need to cross-compile with `#if` guards.  If that is the case, you'll need to include `net452` as a target.  The section on [cross-compiling](#how-to-cross-compile-for-net-core-and-net-framework) contains examples of this.
+
+You can now build the project and see the produced artifacts.
+
+Using DNX:
+
+```
+$ dnu restore
+$ dnu build
+```
+
+Using .NET CLI:
+
+```
+$ dotnet restore
+$ dotnet compile
+```
+
+You will now notice three new entries in your `/bin` folder:
+
+```
+/bin
+   /Debug
+      /net40
+      /dotnet55
+```
+
+**NOTE:** This assumes your code will compile across *both* .NET Core and .NET Framework.  Read the section on [cross-compiling](#how-to-cross-compile-for-net-core-and-net-framework) for how to compile the same file differently for each target if you are using features which are unavailable in some of your targets.
+
+## How to target a Portable Class Library (PCL)
+
+**NOTE:** This requires Windows and .NET Framework installed on your machine.  Refer to the [Prerequisites](#prerequisites) to get dependencies installed.
+
+Targeting a PCL profile is a bit trickier.  For starters, [reference this list of PCL profiles](http://embed.plnkr.co/03ck2dCtnJogBKHJ9EjY/preview) to make sure you have the correct target.  Hover over the name of each entry for the framework identifier, which you will need.
+
+You will then need to do two things:
+
+1. List the dependencies for each target *inside* of that target.  The "global" `dependencies` section cannot be used.
+2. List the framework assemblies you are using in your code *inside* the PCL target profile section.  This will require, at a minimum, `mscorlib`, `System`, and `System.Core`.
```

Comment:
Do you mean that what you'll have to specify will depend on the PCL Profile?  If so, then yes.  I could put that in there somewhere.  Unfortunately the only way I've found to do anything other than a Hello World scenario is to see if a library is supported, find the name of that assembly, and include it then.  I think that's worth mentioning.